redisson/src/main/java/org/redisson/api/MapOptions.java

```diff
@@ -159,7 +165,25 @@ public MapOptions<K, V> writeMode(WriteMode writeMode) {
     public WriteMode getWriteMode() {
         return writeMode;
     }
-    
+
+    public int getWriterRetryAttempts() {
+        return writerRetryAttempts;
+    }
+
+    public MapOptions<K, V> writerRetryAttempts(int writerRetryAttempts) {
+        this.writerRetryAttempts = writerRetryAttempts;
+        return this;
+    }
+
+    public long getWriterRetryInterval() {
+        return writerRetryInterval;
+    }
+
+    public MapOptions<K, V> writerRetryInterval(long writerRetryInterval, TimeUnit timeUnit) {
```

Comment:
Please add comments

Reply:
Done.

Reply:
Please change writerRetryInterval type to Duration